src/app/dev/platforms/desktop/DevToys.Linux/Components/BlazorWebView/BlazorWebViewOptions.cs

```diff
@@ -6,7 +8,7 @@ internal record BlazorWebViewOptions
 
     internal string HostPath { get; init; } = Path.Combine("wwwroot", "index.html");
 
-    internal string ContentRoot => Path.GetDirectoryName(Path.GetFullPath(HostPath))!;
+    internal string ContentRoot { get; init; } = Path.GetDirectoryName(Assembly.GetEntryAssembly()!.Location)!;
```

Comment:
This fix the bug I was talking about in the description where we were looking at `wwwroot/index.html` from the wrong root folder.